### PR TITLE
Carolina

### PR DESCRIPTION
**Pull request by carolarruda**

### CODE REVIEW

File: .bashrc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# git add and commit
```
Comment (procrastian) [inline .bashrc:1]: Not sure that you meant for this file to be part of this repo?

File: spec/cohortManager.spec.js [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+    const expected = {
+      id: 1,
+      cohortName: 'cohort10',
+      students: []
+    }
+    const result = app.addCohort('cohort10')
+    expect(result).toEqual(expected)
+  })
+  it('try to add a new cohort without a name', () => {
+    const result = () => app.addCohort('')
```
Comment (procrastian) [inline spec/cohortManager.spec.js:20]: Could you explain why is this being called as a function? Would having res = app.addCohort(' ') work the same?

File: spec/cohortManager.spec.js [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+      expect(result[i].students[0].lastName).toEqual(
+        expected[i].students[0].lastName
+      )
+      expect(result[i].students[0].githubUsername).toEqual(
+        expected[i].students[0].githubUsername
+      )
+      expect(result[i].students[0].email).toEqual(expected[i].students[0].email)
+    }
+
+    expect(result.students[0].email).toEqual(expected.students[0].email)
```
Comment (procrastian) [inline spec/cohortManager.spec.js:85]: I like how you've laid out multiple expects for the same test! Nice way to tidy up the code!

File: spec/cohortManager.spec.js [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+  it('error message when student name does not exist', () => {
+    app.addCohort('cohort05')
+    app.addStudent('cohort05', 'carolina', 'arruda')
+    const resultTwo = () => app.searchByStudentId('123')
+    expect(resultTwo).toThrowError('Student was not found')
+  })
+  it('error message when adding more students than cohort capacity', () => {
+    app.addCohort('cohort05')
+
+    for (let i = 0; i < 24; i++) {
```
Comment (procrastian) [inline spec/cohortManager.spec.js:211]: Nice way of constructing this test!

File: src/student.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+class Student {
+  constructor(firstName, lastName, githubUsername, email) {
+    this.firstName = firstName
+    this.lastName = lastName
+    this.githubUsername = githubUsername
+    this.email = email
+    this.studentID = this.generateStudentId(7)
+  }
+
+  generateStudentId() {
```
Comment (procrastian) [inline src/student.js:10]: Nice way of generating IDs!

File: src/cohortManager.js [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+  }
+
+  textRemove(githubUsername) {
+    const message = `Hey there, the user with the GitHub username "${githubUsername}" was deleted successfully.`
+    return client.messages
+      .create({
+        body: message,
+        from: '+12178338475',
+        to: '+351912121304'
+      })
```
Comment (procrastian) [inline src/cohortManager.js:166]: Really clean and readable layout of your code! Awesome job!

File: .bashrc (hunk 1 above)
Comment (carolarruda) [reply .bashrc:1]: I was testing shortcuts that Todd created. This allows me use gacp to add, commit and push all at once

File: spec/cohortManager.spec.js (hunk 2 above)
Comment (carolarruda) [reply spec/cohortManager.spec.js:20]: This is related to the TrowError(). I used this as a reference https://stackoverflow.com/questions/4144686/how-can-i-write-a-test-which-expects-an-error-to-be-thrown-in-jasmine

File: spec/cohortManager.spec.js (hunk 3 above)
Comment (carolarruda) [reply spec/cohortManager.spec.js:85]: This was a workaround for the unpredictability of my Ids tbh

**Pull request closed without merging**